iacrcc/convertiacrtrans.md

```diff
@@ -0,0 +1,134 @@
+# Migrating from `iacrtrans` to `iacrcc`
+
+Converting a paper from the `iacrtrans` LaTeX style file (as used by the [*IACR Transactions on Symmetric Cryptology*](http://tosc.iacr.org/) and the [*IACR Transactions on Cryptographic Hardware and Embedded Systems*](http://tches.iacr.org/)) means adjusting how some of the meta-data is provided. Concretely, the biggest change involves adjusting the title, author and affiliation information. This guide doesn't cover every detail, and the full documentation is [located here](https://publish.iacr.org/iacrcc).
+
+### Our starting point
+
+Let’s use a slightly adjusted version of the `iacrtrans` [template](https://github.com/Cryptosaurus/iacrtrans/blob/master/template.tex) as an example:
+
+```latex
+\documentclass{iacrtrans}
+\author{Jane Doe\inst{1,2} \and John Doe\inst{1}}
+\institute{
+  Institute A, City, Country, \email{jane@institute}
+  \and
+  Institute B, City, Country, \email{john@institute}
+}
+\title{My New Paper}
+\begin{document}
+\maketitle
+\keywords{Something \and Something else}
+\begin{abstract}
+  In this paper we prove that the One-Time-Pad has perfect security.
+\end{abstract}
+\section{Introduction}
+We use RSA~\cite{RSA78}.
+\bibliographystyle{alpha}
+\bibliography{mybibliography}
+\end{document}
+```
+Where we assume a file mybibliography.bib exists which contains:
+
+```
+@article{RSA78,
+  author    = {Ronald L. Rivest and Adi Shamir and Leonard M. Adleman},
+  title     = {A Method for Obtaining Digital Signatures and Public-Key Cryptosystems},
+  pages     = {120--126},
+  year      = {1978},
+  journal   = {Communications of the {ACM}},
+  volume    = {21},
+  number    = {2},
+  publisher = {ACM New York, NY, USA},
+  doi       = {10.1145/359340.359342},
+}
+```
+
+### Document class
+
+The conversion starts by changing the `iacrtrans` document class to the `iacrcc`
+document class. Simply change
+
+```latex
+\documentclass{iacrtrans}
+```
+to
+```latex
+\documentclass{iacrcc}
+```
+
+### Title
+
+The title and related information should be provided in the same way (although `iacrcc` supports various optional parameters). If macros (or math) are used in the title then make sure to provide a plaintext version using the `plaintext` option on `\title`.
+
+### Author and affiliations
+
+Author and affiliation information are provided one at-a-time. The `iacrtrans` code
+
+```latex
+\author{Jane Doe\inst{1,2} \and John Doe\inst{1}}
+\institute{
+  Institute A, City, Country, \email{jane@institute}
+  \and
+  Institute B, City, Country, \email{john@institute}
```

Comment:
A lot of people on TCHES use the orcidlink package and then put `\orcid{0000-1111-2222-3333}`. Since we want to encourage people to submit orcids, we might as well include this in the example. Just add `\usepackage{orcidlink}` and then put `\orcid{0000-1111-2222-3333}` after the \inst for an author.

Reply:
Fixed